examples/mnist/quantized.cpp

```diff
@@ -30,10 +33,11 @@
 using namespace tiny_dnn;
 using namespace tiny_dnn::activation;
 
-void construct_net(network<sequential>& nn) {
-    // connection table [Y.Lecun, 1998 Table.1]
+void construct_net(network<sequential> &nn) {
+// connection table [Y.Lecun, 1998 Table.1]
 #define O true
 #define X false
+    // clang-format off
```

Comment:
Please check this thread.
https://github.com/tiny-dnn/tiny-dnn/pull/363#pullrequestreview-5714149

Are we going to add tons of
```cpp
// clang-format off
...
// clang-format on
```
comments? Just because we want to apply auto code formatting?
I guess the effort to have the unified style will end up adding dirty code.